### PR TITLE
[Hub Generated] Review request for Microsoft.ImportExport to add version stable/2020-08-01

### PR DESCRIPTION
**Pull request by nivats**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/nivats/azure-rest-api-specs/dev-storageimportexport-Microsoft.ImportExport-2020-08-01...master/).

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/storageimportexport/resource-manager/Microsoft.ImportExport/stable/2020-08-01/storageimportexport.json [hunk 1]
```diff
@@ -202,7 +203,8 @@
             "in": "query",
             "description": "An integer value that specifies how many jobs at most should be returned. The value cannot exceed 100.",
             "required": false,
-            "type": "integer"
+            "type": "integer",
+            "format": "int64"
```
Comment (nivats) [inline specification/storageimportexport/resource-manager/Microsoft.ImportExport/stable/2020-08-01/storageimportexport.json:207]: Hi @ChenTanyi , This change will not impact customers as we are just adding the int format to fix linter error.

File: specification/storageimportexport/resource-manager/Microsoft.ImportExport/stable/2020-08-01/storageimportexport.json [hunk 2]
```diff
@@ -1166,6 +1169,7 @@
         },
         "percentComplete": {
           "type": "integer",
+          "format": "int64",
```
Comment (nivats) [inline specification/storageimportexport/resource-manager/Microsoft.ImportExport/stable/2020-08-01/storageimportexport.json:1172]: Hi @ChenTanyi , This change will not impact customers as we are just adding the int format to fix linter error.

**Comment by ChenTanyi** [comment]
> Hi @nivats, one or multiple breaking change(s) is detected in your PR. Please check out the breaking change(s), and provide business justification in the PR comment and @ PR assignee why you must have these change(s), and how external customer impact can be mitigated. Please ensure to follow [breaking change policy](https://aka.ms/AzBreakingChangesPolicy) to request breaking change review and approval before proceeding swagger PR review.
> **Action**: To initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://msazure.visualstudio.com/One/_workitems/create/Scenario?templateId=d1c215ec-5d6a-4cb3-a193-95ddd0a1ba96&ownerId=85bb92fb-21fe-41ca-9309-47f2c5f57515). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
> If you think it is false positive breaking change, please provide the reasons in the PR comment, report to Swagger Tooling Team via https://aka.ma/swaggerfeedback.

@nivats You could tell the Breaking Change reviewer about it. I'm not the reviewer for breaking change.

Since most of the SDK may default resolve int as `int32`, it would be breaking change when re-generating. (Maybe it would be safe for using `int32` rather then `int64`, @jianyexi Is it fine for `int32` as non breaking change?)

**Comment by jianyexi** [comment]
> > Hi @nivats, one or multiple breaking change(s) is detected in your PR. Please check out the breaking change(s), and provide business justification in the PR comment and @ PR assignee why you must have these change(s), and how external customer impact can be mitigated. Please ensure to follow [breaking change policy](https://aka.ms/AzBreakingChangesPolicy) to request breaking change review and approval before proceeding swagger PR review.
> > **Action**: To initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://msazure.visualstudio.com/One/_workitems/create/Scenario?templateId=d1c215ec-5d6a-4cb3-a193-95ddd0a1ba96&ownerId=85bb92fb-21fe-41ca-9309-47f2c5f57515). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
> > If you think it is false positive breaking change, please provide the reasons in the PR comment, report to Swagger Tooling Team via https://aka.ma/swaggerfeedback.
> 
> @nivats You could tell the Breaking Change reviewer about it. I'm not the reviewer for breaking change.
> 
> Since most of the SDK may default resolve int as `int32`, it would be breaking change when re-generating. (Maybe it would be safe for using `int32` rather then `int64`, @jianyexi Is it fine for `int32` as non breaking change?)

 Right , for csharp , java , go , the default format is "int32" .

**Pull request closed without merging**